test/test_torch.py

```diff
@@ -13000,6 +13160,25 @@ def test_copy_mem_overlap(self, device):
         self.unary_check_input_output_mem_overlap(
             doubles, sz, lambda input, out: out.copy_(input))
 
+    @torchtest.for_all_device_types()
+    def test_pow_mem_overlap(self, device):
```

Comment:
Why do you still need this? Hasn't this been tested already in `test_binary_op`?

Reply:
`test_binary_op` tests tensor^tensor only and here I test tensor^scalar and scalar^tensor cases

Reply:
renamed